src/test/java/org/fcrepo/client/integration/FcrepoClientIT.java

```diff
@@ -232,8 +249,24 @@ public void testPutLenient() throws Exception {
     @Test
     public void testPatch() throws Exception {
         // Create object
+        create();
+
+        final InputStream body = new ByteArrayInputStream(sparqlUpdate.getBytes());
+
+        // Update triples with sparql update
+        final FcrepoResponse response = client.patch(url)
+                .body(body)
+                .perform();
+
+        assertEquals(NO_CONTENT.getStatusCode(), response.getStatusCode());
+    }
+
+    @Test
+    public void testPatchEtagUpdated() throws Exception {
+        // Create object
         final FcrepoResponse createResp = create();
-        final String createdEtag = createResp.getHeaderValue(ETAG);
+        String createdEtag = createResp.getHeaderValue(ETAG);
+        createdEtag = createdEtag.substring(2, createdEtag.length());
```

Comment:
That's a good suggestion, I wasn't aware of that helper.  Thanks!  However, it appears that tag.getValue() returns `d51947bb496af13188617d9abf702ce99c30efa8` rather than `"d51947bb496af13188617d9abf702ce99c30efa8"`, and fcrepo requires the quotes.

Subtracting the quotes and then adding them back feels a little awkward.  Although if I change If-Match to add back in quotes it would feel a bit more generalizable.  Hmm...